scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -53,7 +53,8 @@ check-transaction-versions:
     - npm install --ignore-scripts -g @polkadot/metadata-cmp
     # Set git config
     - git config remote.origin.url "https://github.com/paritytech/polkadot.git"
-    - git fetch origin release
+    - export LATEST_RELEASE=$(curl --silent "https://api.github.com/repos/paritytech/polkadot/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
use `jq -e -r '.tag_name'` to extract the tag name

also need to use the `GITHUB_TOKEN` in the `Authorization` header or your request will be rate-limited by the GitHub API

everything put together:

`    - export LATEST_RELEASE=$(curl -sSLf  -H "Authorization: token $GITHUB_TOKEN" "https://api.github.com/repos/paritytech/polkadot/releases/latest" | jq -e -r '.tag_name')`